src/Plugin/Block/LangDropDownSwitcher.php

```diff
@@ -19,7 +19,7 @@
 use Drupal\Core\Form\FormBuilderInterface;
 use Drupal\user\Entity\Role;
 use Symfony\Component\DependencyInjection\ContainerInterface;
-use Symfony\Component\HttpFoundation\File\MimeType\MimeTypeGuesserInterface;
+use Drupal\Core\ProxyClass\File\MimeType\MimeTypeGuesser;
```

Comment:
This fix is wrong, never reference to proxy, please check core's changes about the class

Reply:
The CR is for 9.1 core https://www.drupal.org/node/3126004

Reply:
Also please provide PR description

Reply:
Updated change.

Reply:
I see no `guessMimeType()` - did you read change record? Also I updated for you PR description